packages/contracts/test/router/v6/sdk.test.ts

```diff
@@ -44,10 +46,300 @@ describe("[ReservoirV6_0_0] Filling listings and bids via the SDK", () => {
+      });
+    }
+
+    // Order 3: Seaport WETH
+    const seller3 = carol;
+    const tokenId3 = 2;
+    const price3 = parseEther("0.11");
+    const fee3 = bn(1120);
+    {
+      // Mint erc721 to seller
+      await erc721.connect(seller3).mint(tokenId3);
+
+      // Approve the exchange
+      await erc721
+        .connect(seller3)
+        .setApprovalForAll(Sdk.Seaport.Addresses.Exchange[chainId], true);
+
+      // Build sell order
+      const builder = new Sdk.Seaport.Builders.SingleToken(chainId);
+      const sellOrder = builder.build({
+        side: "sell",
+        tokenKind: "erc721",
+        offerer: seller3.address,
+        contract: erc721.address,
+        tokenId: tokenId3,
+        paymentToken: Sdk.Common.Addresses.Weth[chainId],
+        price: price3.sub(price3.mul(fee3).div(10000)),
+        fees: [
+          {
+            amount: price3.mul(fee3).div(10000),
+            recipient: deployer.address,
+          },
+        ],
+        counter: 0,
+        startTime: await getCurrentTimestamp(ethers.provider),
+        endTime: (await getCurrentTimestamp(ethers.provider)) + 60,
+      });
+      await sellOrder.sign(seller3);
+
+      await sellOrder.checkFillability(ethers.provider);
+
+      // listings.push({
+      //   kind: "seaport",
+      //   contractKind: "erc721",
+      //   contract: erc721.address,
+      //   tokenId: tokenId3.toString(),
+      //   order: sellOrder,
+      //   currency: Sdk.Common.Addresses.Weth[chainId],
+      // });
+    }
+
+    const usdc = new Sdk.Common.Helpers.Erc20(
+      ethers.provider,
+      Sdk.Common.Addresses.Usdc[chainId]
+    );
+    const weth = new Sdk.Common.Helpers.Weth(ethers.provider, chainId);
+
+    const seller1EthBalanceBefore = await seller1.getBalance();
+    const seller2UsdcBalanceBefore = await usdc.getBalance(seller2.address);
+    const seller3WethBalanceBefore = await weth.getBalance(seller3.address);
+    const token1OwnerBefore = await erc721.ownerOf(tokenId1);
+    const token2OwnerBefore = await erc721.ownerOf(tokenId2);
+    const token3OwnerBefore = await erc721.ownerOf(tokenId3);
+
+    expect(token1OwnerBefore).to.eq(seller1.address);
+    expect(token2OwnerBefore).to.eq(seller2.address);
+    // expect(token3OwnerBefore).to.eq(seller3.address);
+
+    const router = new Sdk.RouterV6.Router(chainId, ethers.provider);
+    router.contracts.permit2Module = permit2Module;
+    const tx = await router.fillListingsTx(
+      listings,
+      buyer.address,
+      Sdk.Common.Addresses.Usdc[chainId],
+      {
+        source: "reservoir.market",
+      }
+    );
+
+    // Trigger approvals
+    for (const approval of tx.approvals) {
+      await buyer.sendTransaction(approval.txData);
+    }
+
+    const permitHandler = new Permit2.Handler(chainId, ethers.provider, permit2Module.address);
+    
+    // Sign permits
+    for (const permit of tx.permits) {
+      // Override permit start and end times
```

Comment:
Don't we need to update the permit expiration? Otherwise the tests will only succeed if ran on a very recent block.